modules/helm-installing-helm.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id='installing-helm_{context}']
+= Installing Helm
+
+.Prerequisites:
+* You have installed Go version 1.13.
+
+. To install Helm, download the binary and create the Helm binary in the `/bin` directory with the `make` command:
++
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest
+$ GO111MODULE=on make
+----
+
+. Check the installed version:
++
+----
+$ helm version
+$ version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"a41f66e7be615d8602898e4478dec70e5f60e23c", GitTreeState:"clean", GoVersion:"go1.13.1"}
```

Comment:
Is this line output or a command? If it's output of the "helm version" command, then it shouldn't have "$" in front of it.